src/server/server.cc

```diff
@@ -839,6 +839,11 @@ void Server::cron() {
       cleanupExitedWorkerThreads(false);
     }
 
+    // check if we need to evict connections every 10s
+    if (counter != 0 && counter % 100 == 0 && config_->max_memory_clients > 0) {
```

Comment:
Can we make the condition `config_->max_memory_clients > 0` the first one to benefit from fast evaluation if this option is disabled in the config?